File: src/main/java/org/dxworks/githubminer/dto/response/repository/commits/Commit.java
package org.dxworks.githubminer.dto.response.repository.commits;

import com.google.api.client.util.Key;
import lombok.Data;

@Data
public class Commit {
    @Key
    private CommitAuthor author;
    @Key
    private CommitAuthor committer;
    @Key
    private String message;
    @Key
    private String url;
}

File: github-miner-core/src/main/kotlin/org/dxworks/githubminer/dto/response/repository/commits/Commit.kt
package org.dxworks.githubminer.dto.response.repository.commits

import com.google.api.client.util.Key

class Commit {
    @Key
    var author: CommitAuthor? = null

    @Key
    var committer: CommitAuthor? = null

    @Key
    var message: String? = null

    @Key
    var url: String? = null
}
